webpack.config.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+      main: './src/index.js',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
```

Comment:
See https://webpack.js.org/guides/output-management/ for details on using `HtmlWebpackPlugin`